x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/__tests__/use_monitor_breadcrumbs.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ChromeBreadcrumb } from 'kibana/public';
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { of } from 'rxjs';
+import { MountWithReduxProvider, mountWithRouter } from '../../../../../lib';
+import { KibanaContextProvider } from '../../../../../../../../../src/plugins/kibana_react/public';
+import { useMonitorBreadcrumb } from '../use_monitor_breadcrumb';
+import { OVERVIEW_ROUTE } from '../../../../../../common/constants';
+import { Ping } from '../../../../../../common/runtime_types/ping';
+import { JourneyState } from '../../../../../state/reducers/journey';
+
+describe('useMonitorBreadcrumbs', () => {
+  it('sets the given breadcrumbs', () => {
+    const [getBreadcrumbs, core] = mockCore();
+
+    const Component = () => {
+      useMonitorBreadcrumb({
+        activeStep: { monitor: { id: 'test-monitor' } } as Ping,
+        journey: { details: { timestamp: '2021-01-04T11:25:19.104Z' } } as JourneyState,
+      });
+      return <>Step Water Fall</>;
+    };
+
+    mountWithRouter(
+      <MountWithReduxProvider>
+        <KibanaContextProvider services={{ ...core }}>
+          <Route path={OVERVIEW_ROUTE}>
+            <Component />
+          </Route>
+        </KibanaContextProvider>
+      </MountWithReduxProvider>
+    );
+
+    expect(getBreadcrumbs()).toMatchInlineSnapshot(`
```

Comment:
Thinking about this now, I'm realizing I'm wrong, since this is testing the output of the hook, not a specific render.  Would the breadcrumbs html be rendered at all in this test?

Reply:
so breadcrumb are rendered outside uptime app, so testing via that means we will have to mount the whole app.